src/modules/storage/storage.cpp

```diff
@@ -271,7 +284,29 @@ tuple<int, string, json> Storage::readBlock(int height) {
     }
     ++tx_pos;
   }
+  int mtree_size = (stoi(findBy("block_header", block_id, "txcnt")) * 2) - 1;
+  for (int mtree_index = 0; mtree_index < mtree_size; ++mtree_index) {
+    string mtree = findBy("transaction", block_id, to_string(mtree_index));
+    transaction_json[tx_pos]["mtree"][mtree_index] = mtree;
+  }
   return make_tuple(height, block_binary, transaction_json);
 }
 
+vector<string> Storage::findSibling(const string &tx_id) {
+  string tx_id_pos = findBy("transaction", tx_id, "mPos");
+  string block_id = findBy("transaction", tx_id, "bID");
+  string tx_cnt = findBy("block_header", block_id, "txcnt");
+  int iteration_size = int(log2(stoi(tx_cnt)));
+
+  vector<string> siblings;
+  int tx_id_pos_int = stoi(tx_id_pos);
+  for (unsigned int i = 0; i < iteration_size; ++i) {
+    int mtree_pos =
+        (tx_id_pos_int % 2 != 0) ? tx_id_pos_int - 1 : tx_id_pos_int + 1;
+    siblings.push_back(findBy("transaction", block_id, to_string(mtree_pos)));
```

Comment:
```suggestion
    siblings.emplace_back(findBy("transaction", block_id, to_string(mtree_pos)));
```
emplace_back 으로 해야 성능에 조금 더 좋습니다.